src/cordova/android/OutlineAndroidLib/outline/build.gradle

```diff
@@ -1,17 +1,18 @@
 plugins {
     id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
 }
 
 // Needed for the composite build to recognize this module as 'org.outline:outline:0.0'.
 group = 'org.outline'
 
 android {
     namespace 'org.outline'
-    compileSdk 32
+    compileSdk 33
 
     defaultConfig {
         minSdk 22
-        targetSdk 32
+        targetSdk 33
```

Comment:
I don't think we can accept this change without ensuring our app works well with API 33.
Do you need this change? I believe you can set a different target SDK on your app.

Reply:
Yes, I can set the SDK to 32. 

I made this change is because since Android Studio Electric Eel ( 2022.1.1 Patch 1), the initial compileSdk and targetSdk are 33 while starting a new project. I want developers no need to downgrade the SDK version after they create a new project.

So we keep the SDK version to 32?